react-native-libraries.json

```diff
@@ -9618,5 +9618,10 @@
     "examples": ["https://github.com/vonovak/react-native-theme-control/tree/main/example"],
     "ios": true,
     "android": true
+  },
+  {
+    "githubUrl": "https://github.com/neo773/react-native-macos-fast-image",
+    "npmPkg": "react-native-macos-fast-image",
```

Comment:
Hello @neo773, thank you for adding new package to the directory! 👍 

```suggestion
```

A small nit before we ship the addition - `npmPkg` can be skipped in your package case.

Reply:
Since I have made conflicts in this PR by merging the other one decided to resolve them by myself, and applied the suggested change in the process, so PR whould be ready to ship. Feel free to prepare any correction if I missed or messed something up! 🙂